components/kyma-operator/pkg/kymahelm/kyma-helm-client.go

```diff
@@ -71,6 +75,37 @@ func (hc *Client) ReleaseStatus(rname string) (string, error) {
 	return strings.Replace(statusStr, `\n`, "\n", -1), nil
 }
 
+//IsReleaseDeletable returns true for release that can be deleted
+func (hc *Client) IsReleaseDeletable(rname string) (bool, error) {
```

Comment:
Nit: consider that in the future this method will be also used in upgrade scenarios which will have different predicate determining if release is ok for delete.

Reply:
i would go with different method for upgrade rather